### ISSUE TITLE
The client GUI doesn't launch on arm64 device.

### ISSUE THREAD

**Issue by kreaninw** [pia-foss/desktop#28]
**Describe the bug**
I successfully build the client for arm64 device. And I am able to install the build with no error as shown below:

![Screenshot 2021-06-11 142425](https://user-images.githubusercontent.com/1403194/121736683-e0928980-cac5-11eb-9013-ade83ceb511e.jpg)

However, when I tried to launch the client, nothing happened. Then, I checked the system monitor but **_pia-client_** is not running as it should.

I can still use the `piactl` commands though.

**To Reproduce**
1. Download the installer for arm64 device [here](https://drive.google.com/file/d/1J0v3tnG_WVSIBncEdTsEkos4a2aq5dd7/view?usp=sharing). It is my build following the method in this repo.
2. Navigate to the download location and install the client in the terminal with `sh pia-linux-arm64-2.9-00032.debug.run`.
3. Open the client.
4. The GUI will not launch and there's no `pia-client` process running.

**Expected behavior**
The client's GUI should launch and the pia-client process should be running. 

**Observed on:**
 - OS: Ubuntu Desktop 64-bit for Raspberry Pi 4
 - Version: 21.04
 - Device: Raspberry Pi 4 model B, 8GB RAM

**Comment by JonathonH-PIA** [comment]
Hi @kreaninw ,

Do you get the same behavior when running our arm64 builds from the releases page?  (Try 2.9 or 2.10-beta.1) https://github.com/pia-foss/desktop/releases

That'll indicate whether it's an issue specific to this build (possibly different build deps), or whether it's a general issue affecting this distribution, and we can troubleshoot from there.

Do you get any diagnostic output when running `/opt/piavpn/bin/pia-client` from the terminal on your build?

**Comment by kreaninw** [comment]
> Hi @kreaninw ,
> 
> Do you get the same behavior when running our arm64 builds from the releases page? (Try 2.9 or 2.10-beta.1) https://github.com/pia-foss/desktop/releases
> 
> That'll indicate whether it's an issue specific to this build (possibly different build deps), or whether it's a general issue affecting this distribution, and we can troubleshoot from there.
> 
> Do you get any diagnostic output when running `/opt/piavpn/bin/pia-client` from the terminal on your build?

**Version 2.9:** both my build and your build don't launch. This is the output I get when running `/opt/piavpn/bin/pia-client` from the terminal:

```
$ ./pia-client
[2021-06-12 00:55:47.466][53f2][json.settings][common/src/json.cpp:327][warning] Unable to read from clientsettings.json
[2021-06-12 00:55:47.467][53f2][default][client/src/linux/linux_env.cpp:21][info] XDG_CURRENT_DESKTOP= "ubuntu:GNOME"
[2021-06-12 00:55:47.468][53f2][default][client/src/linux/linux_env.cpp:69][info] Detected desktop "GNOME" and RTL: false
[2021-06-12 00:55:47.468][53f2][default][info] Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
[2021-06-12 00:55:47.480][53f2][qt.qpa.plugin][info] Could not load the Qt platform plugin "xcb" in "" even though it was found.
[2021-06-12 00:55:47.480][53f2][default][fatal] This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.
[2021-06-12 00:55:47.480][53f2][default][fatal] 
[2021-06-12 00:55:47.480][53f2][default][fatal] Available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, wayland-egl, wayland, webgl, xcb.
[2021-06-12 00:55:47.480][53f2][default][fatal] 
Aborted (core dumped)
```

**Version 2.10 beta:** It seems this issue has already been fixed as I can launch the application's GUI just fine as shown below:

![SharedScreenshot](https://user-images.githubusercontent.com/1403194/121760793-86f88200-cafa-11eb-8dda-8830bf9c7edb.jpg)

I will test the connection in 2.10 again when I get home (I am remotely controlling the device through the manual connection). However, that would be a totally different issue.

Thanks.

**Comment by JonathonH-PIA** [comment]
Thanks @kreaninw !  That was due to a mistake in the arm64 dependencies directory, the built libxcb libraries were in an artifacts subdirectory by mistake (https://github.com/pia-foss/desktop/tree/master/deps/built/linux/arm64 - those come from the desktop-dep-build repository).  As a result, PIA used the host libxcb, which is compatible on most distributions, but apparently not on 21.04.

The fix in 2.10-beta.1 is up on the `beta` branch if you want to build from source with the fix.  Glad to hear this is working for you now!
